nft_ingester/src/message_parser.rs

```diff
@@ -193,6 +194,7 @@ impl MessageParser {
                     slot_updated: account_update.slot as i64,
                     amount: ta.amount as i64,
                     write_version: account_update.write_version,
+                    data_hash: calc_solana_account_data_hash(&account_update.data),
```

Comment:
Could we approximately calculate the performance impact of calculating hashes "in place" for each account? We have many account updates, way larger updates amount than we have for transactions, so it would be great to understand if this increases account processing time

Reply:
According to my tests, it takes ~ 9 microseconds to calculate hash for 1KB of data, which looks acceptable.
(anyway xxhash is one of most performant hashing algorithms)